DEVELOPERS.md

```diff
@@ -2,44 +2,46 @@
 
 1. [Development setup](#development-setup)
     - [Install dependencies](#install-dependencies)
-    - [Fork the repository](#fork-the-repository)
-1. [Build Supabase](#build-supabase)
-    - [Choose a directory](#choose-a-directory)
-1. [Start a development server](#start-a-development-server)
-    - [Supabase Website](#supabase-website)
-    - [Supabase Docs](#supabase-docs)
-    - [Supabase Studio](#supabase-studio)
-1. [Create a pull request](#create-a-pull-request)
+2. [Local development](#local-development)
+   - [Fork the repository](#fork-the-repository)
+   - [Clone the repo](#clone-the-repo)
+   - [Running turborepo](#running-turborepo)
+      - [Shared components](#shared-components)
+      - [Installing packages](#installing-packages)    
+3. [Create a pull request](#create-a-pull-request)
 
 - [Common tasks](#common-tasks)
   - [Add a redirect](#add-a-redirect)
-- [Monorepo](#monorepo)
-  - [Getting started](#getting-started)
-  - [Shared components](#shared-components)
-  - [Installing packages](#installing-packages)
-  - [Development](#development)
 - [Community channels](#community-channels)
 
-## Development setup
+## Getting started
 
-Thanks for your interest in Supabase and for wanting to contribute! Before you begin, read the
+Thanks for your interest in [Supabase](https://supabase.com) and for wanting to contribute! Before you begin, read the
 [code of conduct](https://github.com/supabase/.github/blob/main/CODE_OF_CONDUCT.md) and check out the
 [existing issues](https://github.com/supabase/supabase/issues).
-This document describes how to set up your development environment to build and test Supabase.
+This document describes how to set up your development environment to build and test [Supabase](https://supabase.com).
 
 ### Install dependencies
 
-You need to install and configure the following dependencies on your machine to build Supabase:
+You need to install and configure the following dependencies on your machine to build [Supabase](https://supabase.com):
 
 - [Git](http://git-scm.com/)
 - [Node.js v16.x (LTS)](http://nodejs.org)
 - [npm](https://www.npmjs.com/) version 7+ or [Yarn](https://yarnpkg.com/)
 
+## Local development
+
+We are in the process of migrating this repository to monorepo, using [Turborepo](https://turborepo.org/docs).
```

Comment:
Is this still relevant? Looks like we're all-in on Turbo now?

Reply:
I think docs is the last holdout? 

<img width="294" alt="CleanShot 2022-09-23 at 11 02 00@2x" src="https://user-images.githubusercontent.com/105593/191971651-e84a82c7-e068-47b2-8542-33a7f577ece1.png">

Reply:
When that changes, lets adjust our docs to be turborepo focussed ?